### PR TITLE
Optimize bubbe sort

### PR DESCRIPTION
**Pull request by VARoDeK**
- as after each step of 'outer-loop', one number reaches to its right position. Thus inner loop does not need to run for 'n' times in every cycle of 'outer-loop'.

(Put an `X` inside the `[ ]` to denote check mark `[X]`.)

- [ ] **If creating a new file :**
  - [ ] added links to it in the README files ?
  - [ ] included tests with it ?
  - [ ] added description (overview of algorithm, time and space compleixty, and possible edge case) in docstrings ?

- [ x] **if done some changes :**
  - [ x] wrote short description in the PR explaining what the changes do ?
  - [ x] Fixes #[issue number] if related to any issue

- [ ] **other**

### CODE REVIEW

File: algorithms/sort/bubble_sort.py [hunk 1]
```diff
@@ -20,10 +20,11 @@ def swap(i, j):
     iteration = 0
     if simulation:
         print("iteration",iteration,":",*arr)
-    
+    x=-1
```
Comment (goswami-rahul) [inline algorithms/sort/bubble_sort.py]: please use spaces before and after `=` following PEP guide.

File: algorithms/sort/bubble_sort.py [hunk 2]
```diff
@@ -20,10 +20,11 @@ def swap(i, j):
...
     while swapped:
         swapped = False
-        for i in range(1, n):
+        x = x+1
+        for i in range(1, n-x):
```
Comment (goswami-rahul) [inline algorithms/sort/bubble_sort.py]: give spaces here too.

File: algorithms/sort/bubble_sort.py (hunk 1 above)
Comment (VARoDeK) [reply algorithms/sort/bubble_sort.py]: Okay!

File: algorithms/sort/bubble_sort.py (hunk 2 above)
Comment (VARoDeK) [reply algorithms/sort/bubble_sort.py]: Doing it!

Comment (goswami-rahul) [reply algorithms/sort/bubble_sort.py]: Looks good :+1:

**Pull request merged**